src/User32.Desktop/User32.cs

```diff
@@ -109,6 +128,12 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, StringBuilder lParam);
+
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, uint wParam, int lParam);
```

Comment:
No, I meant UTF8 and UTF16. But ANSI is a fine example as well. Basically if all your method takes is a pointer to a buffer, that buffer might contain text with any of dozens of encodings. But UTF8 and UTF16 are the most common ones in .NET at least, it seems.
Didn't we already discuss the merits of various charset values and decide that .NET uses UTF16 on all operating systems after Windows 95 anyway? So is Auto really interesting or should we just use Unicode?